base/libgit2/libgit2.jl

```diff
@@ -219,9 +219,9 @@ function branch!(repo::GitRepo, branch_name::AbstractString,
                  force::Bool=false,           # force branch creation
                  set_head::Bool=true)         # set as head reference on exit
     # try to lookup branch first
-    branch_ref = Nullable{GitReference}(force ? nothing : lookup_branch(repo, branch_name))
+    branch_ref = force ? Nullable{GitReference}() : lookup_branch(repo, branch_name)
```

Comment:
this looks better than relying on the conversion from `nothing` to a null anyway IMO